app/collections/tracks.js

```diff
@@ -1,24 +1,49 @@
 import Backbone from 'backbone';
 import Track from '../models/track';
+import cozysdk from 'cozysdk-client';
 
 
 const Tracks = Backbone.Collection.extend({
     model: Track,
+
+    initialize: function(models, options) {
+        this.on('add', this.onAdd, this);
+        if (options) {
+            this.type = options.type;
+        }
+        this._attributes = {}
```

Comment:
I don't understand what's your `_attributes`. A collection can't have any attributes: it's just a list of elements. If it has, then it's more a model with a collection relied to. What's the purpose of this one?